backend/prisma/migrations/20221106065039_add_initial_accounts/migration.sql

```diff
@@ -0,0 +1,31 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
```

Comment:
this allows us to use the `uuid_generate_v4()` function to generate uuids in postgres